@xen-orchestra/lite/src/components/RangeInput.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+interface ParentState {}
+
+interface State {
+  currentValue: number
+  input: React.RefObject<HTMLInputElement>
+}
+
+interface Props {
+  defaultValue: number
+  max: number
+  min: number
+  onChange: (value: number) => void
+  step: number
+}
+
+interface ParentEffects {}
+
+interface Effects {
+  _onChange: () => void
+}
+
+interface Computed {}
+
+const RangeInput = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    initialState: props => ({
+      currentValue: props.defaultValue,
```

Comment:
Make it a controlled component instead of handling a default value.

Reply:
I don't understand. He is already a controlled component no ? 
The value of the input are equal to the value state `currentValue`.
The `defaultValue` prop are here just to set a first value.

Reply:
I'm talking about `RangeInput`. Right now, `input` is controlled by `RangeInput` but `RangeInput` isn't controlled by its parent.